drwxr-xr-x@  5 bilibili  staff   160B Aug 21 11:19 client
diff --git a/src/com/bible/bible.go b/src/com/bible/bible.go
index caef1fd..f19f605 100644
--- a/src/com/bible/bible.go
+++ b/src/com/bible/bible.go
@@ -2,6 +2,7 @@ package main

 import (
        "fmt"
+       "net/http"
        "os"
        "time"
 )
@@ -43,11 +44,11 @@ func main() {
        go fibonacci(cap(c), c)
        fmt.Println("test")

-       c = make(chan int, 1)
-       for i := 0; i< 5; i++ {
-               c <- i
-               TestReturn(c)
-       }
+       //c := make(chan int, 1)
+       //for i := 0; i< 5; i++ {
+       //      c <- i
+       //      TestReturn(c)
+       //}

        for {
                fmt.Println(os.Getenv("USER"))