diff --git a/src/com/bible/constants/constants.go b/src/com/bible/constants/constants.go
index dc7adb6..2aca35a 100644
--- a/src/com/bible/constants/constants.go
+++ b/src/com/bible/constants/constants.go
@@ -3,11 +3,7 @@ package constants
 const Name = "veitch"

 func Add(x int, y int) int {
-       if x > 3 {
-               return x - y
-       } else {
-               return x + y
-       }
+       return x + y
 }

 func reduce(before int, to int) int {
